src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -44,6 +44,30 @@ const AnalysisContext = React.createContext(initialContext);
 function AnalysisProvider(props) {
   const [analysisContext, setAnalysisContext] = useState(initialContext);
 
+  // Update the analysis details that
+  // are required to display the progression
+  // using polling
+  useEffect(() => {
```

Comment:
I am wondering if we should not make this into a React hook (the polling portion) or if there is an existing one we can use.  This seems like common logic.

Reply:
You could look at this: https://blog.bitsrc.io/polling-in-react-using-the-useinterval-custom-hook-e2bcefda4197

Reply:
Updated in a48aa588b48496b26293670142239f3ee580ef13